### PR TITLE
Removed table and added community block to the index view.

### LINKED ISSUE #97: Update community list columns
Currently we should only be showing the Name and number of AWRVI Index dashboards.

### PR DESCRIPTION
**Pull request by grimm**
Fixes #97

### CODE REVIEW

**Comment by grimm** [comment]
Ready for review

File: app/views/communities/index.html.haml [hunk 1]
```diff
@@ -4,22 +4,9 @@
...
-        %tr
-          %td= community.name
-          %td= community.gnis_id
-          %td= community.location
-          %td= link_to 'Show', community
-          %td= link_to 'Edit', edit_community_path(community)
-          %td= link_to 'Destroy', community, method: :delete, data: { confirm: 'Are you sure?' }
+  .panel-body
+    - @communities.each do |community|
+      = render 'community', community: community
```
Comment (teknofire) [inline app/views/communities/index.html.haml]: This should just be `= render community`

**Comment by grimm** [comment]
Ready for review.

**Comment by teknofire** [comment]
This is more what I had in mind for the initial layout, still needs to have the link added to view the community

awrvi

**Comment by vlraymond** [comment]
Looks awesome, I like it

**Comment by grimm** [comment]
So what else needs to be done with this pull request?

**Pull request merged**